setup.cfg

```diff
@@ -12,10 +12,21 @@ addopts =
     --ignore examples
     --ignore docs
     --doctest-modules
+    --durations 10
+    --timeout 600
+    --cov sktime
+    --cov-report xml
+    --cov-report html
+    --showlocals
+    --pyargs
+    --verbose
+    -n 2
 filterwarnings =
     ignore::UserWarning
     ignore:numpy.dtype size changed
     ignore:numpy.ufunc size changed
+testpaths =
```

Comment:
this switches test discovery from package based to path based, have you verified that the same number of tests are discovered with and without this change?

Reply:
yes I just verified that, its same